### ISSUE TITLE
Modify the table hover attribute

### ISSUE THREAD

**Issue by santaimpersonator** [squidfunk/mkdocs-material#5061]
### Context

Modify the background color of a table row when a mouse hovers over a table's row.

### Description

When using the default dark color palette, the background color change when hovering over a table's row is almost indistinguishable.

![background_color](https://user-images.githubusercontent.com/36016723/219495355-28484bd8-aa30-4b91-956f-56176d793191.gif)
*From the MkDocs-Material [documentation](https://squidfunk.github.io/mkdocs-material/reference/data-tables/?h=table#usage)*

### Related links

- [Data Table documentation](https://squidfunk.github.io/mkdocs-material/reference/data-tables/)
- [Light/Dark Color Palette documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette-toggle)
- [Stylesheet code *(from repo: lines 512-521)*](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/stylesheets/main/_typeset.scss#L512-L521)

### Use Cases

A white or black color won't work with both colr schemes. However, a nuetral color like yellow or green seems to be fine, with a higher alpha value.

Obviously, a new color variable could be created and called to match each of the light and dark color schemes. However, this seemed like a simple solution to the problem. Plus, the information for this modification could be documented as a stylesheet override in the [data table section](https://squidfunk.github.io/mkdocs-material/reference/data-tables/) for users who'd want to customize it.

### Visuals

Yellow Modification:
---
``` css
.md-typeset table:not([class]) tbody tr:hover {
    background-color: rgba(255,255,0,.08);
}
```
![background_green](https://user-images.githubusercontent.com/36016723/219501324-c297d1c4-ae0e-48a3-8d6d-90dc398bbe71.gif)

Green Modification:
---
``` css
.md-typeset table:not([class]) tbody tr:hover {
    background-color: rgba(0,255,0,.08);
}
```

![background_yellow](https://user-images.githubusercontent.com/36016723/219501303-9a3d339c-3438-4e0b-b6e0-79f0e9d453f2.gif)

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material), related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by squidfunk** [comment]
Thanks for reporting. Fixed in 8796d9008. There's now a designated CSS variable which defines the table hover color. The slate color scheme defines a shade that mimics the same intensity as the light scheme now:

<img width="270" alt="Bildschirmfoto 2023-02-18 um 10 56 10" src="https://user-images.githubusercontent.com/932156/219854101-7f6c4d3b-4113-4c80-8d09-a5d38d8cc569.png">

Additionally, if wished, you can now easily override the hover color with [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css).

__For all color schemes__:

``` css
[data-md-color-scheme] {
  --md-typeset-table-color--light: rgba(255, 255, 0, .08);
}
```

__For `slate` color scheme only__:

``` css
[data-md-color-scheme="slate"] {
  --md-typeset-table-color--light: rgba(255, 255, 0, .08);
}
```

**Comment by squidfunk** [comment]
Released as part of [9.0.13](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.13).
